src/h5web/visualizations/heatmap/PixelAxis.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { Axis } from '@vx/axis';
+import { scaleLinear } from 'd3-scale';
+import { format } from 'd3-format';
+import { AxisOrientation } from '@vx/axis/lib/types';
+import styles from './HeatmapVis.module.css';
+
+interface Props {
+  orientation: AxisOrientation;
```

Comment:
You can restrict the type here to a subset, to avoid having to deal with top/right.

Reply:
That is sensible given that those are not implemented (yet?)